layouts/shortcodes/keps-data.html

```diff
@@ -0,0 +1,81 @@
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.12.1/css/jquery.dataTables.css">
```

Comment:
This belongs elsewhere in the layout. Consider using [`.HasShortcode`](https://gohugo.io/templates/shortcode-templates/#checking-for-existence) to make that inclusion conditional, and leave a comment here to signpost folks to where it's done.